.github/workflows/_test.yaml

```diff
@@ -103,18 +103,12 @@ jobs:
           ALL_RUNNERS: |-
             {
               "trusted_builds": {
```

Comment:
Simplifying the existing runners expressions. This does not break existing code using this logic.